join-chat-to-teams-meeting/Chat_Teams_InteropApp.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Chat_Teams_InteropApp.swift
+//  Chat Teams Interop
+//
+//  Copyright (c) Microsoft Corporation.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+@main
+struct Chat_Teams_InteropApp: App {
```

Comment:
Is there a reason we didn't use camel case here?

Reply:
This is the default from Xcode. The quickstart is about going form ground zero, with minimal changes to defaults.